client/cli/src/params.rs

```diff
@@ -476,9 +476,15 @@ pub struct RunCmd {
 	#[structopt(long = "rpc-cors", value_name = "ORIGINS", parse(try_from_str = parse_cors))]
 	pub rpc_cors: Option<Cors>,
 
-	/// Specify Grafana data source server TCP Port.
-	#[structopt(long = "grafana-port", value_name = "PORT")]
-	pub grafana_port: Option<u16>,
+	/// Specify Prometheus endpoint TCP Port.
+	#[structopt(long = "prometheus-port", value_name = "PORT")]
+	pub prometheus_port: Option<u16>,
+
+	/// Disable connecting to the Substrate prometheus.
```

Comment:
```suggestion
	/// Do not expose a Prometheus metric endpoint.
```

Reply:
One does not disable others to connect to it, but the exposition in itself.

Reply:
complete